src/jllvm/materialization/CodeGenerator.cpp

```diff
@@ -1465,21 +1468,21 @@ void CodeGenerator::generateEHDispatch()
     m_builder.SetInsertPoint(continueBlock);
 }
 
-void CodeGenerator::generateNullPointerCheck(llvm::Value* object)
+void CodeGenerator::generateBuiltinExceptionThrow(llvm::Value* condition, llvm::StringRef builderName,
+                                                  llvm::ArrayRef<llvm::Type*> argTypes,
```

Comment:
I don't think the `argTypes` parameter is necassary. Can we deduce the input types simply by taking the types of the values from `args`?